src/proto/flwr/proto/transport.proto

```diff
@@ -44,12 +44,13 @@ message ServerMessage {
     Parameters parameters = 1;
     map<string, Scalar> config = 2;
   }
-
+  message PropertiesIns { map<string, Scalar> properties = 1; }
```

Comment:
```suggestion
  message PropertiesIns { map<string, Scalar> config = 1; }
```

Reply:
@pedropgusmao not that this renaming probably requires a few downstream renamings as well

Reply:
@danieljanes same for `PropertiesRes` ?